configs/next/packages/gcc/sources

```diff
@@ -23,7 +23,7 @@ ATSRC_PACKAGE_NAME="GCC (GNU Compiler Collection)"
 ATSRC_PACKAGE_SUBPACKAGE_NAME_0="GNU Standard C++ Library v3 (Libstdc++-v3)"
 ATSRC_PACKAGE_SUBPACKAGE_NAME_1="GNU Libgomp"
 ATSRC_PACKAGE_VER=7.0.1
-ATSRC_PACKAGE_REV=127080a4e745
+ATSRC_PACKAGE_REV=c1995a329e8b
 ATSRC_PACKAGE_LICENSE="GPL 3.0"
 ATSRC_PACKAGE_DOCLINK="https://gcc.gnu.org/onlinedocs/gcc-6.2.0/gcc/"
 ATSRC_PACKAGE_SUBPACKAGE_DOCLINK_0="https://gcc.gnu.org/libstdc++/"
```

Comment:
The documentation link is outdated. I guess AT next should point to https://gcc.gnu.org/onlinedocs/gcc/, which is the latest documentation available.